src/main/java/bral/citibike/aws/CitiBikeRequestHandler.java

```diff
@@ -8,21 +8,40 @@
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
 import com.google.gson.Gson;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
 
 public class CitiBikeRequestHandler implements RequestHandler<APIGatewayProxyRequestEvent,
         CitiBikeRequestHandler.CitiBikeResponse> {
 
+    private final S3Client s3Client;
+    private final StationsCache stationsCache;
+
+    public CitiBikeRequestHandler()
+    {
+        // Initialize the s3 client
+        //s3Client = S3Client.create();
+        s3Client = S3Client.builder()
+                .region(Region.US_EAST_2) // Specify your AWS region
+                .build();
+        this.stationsCache = new StationsCache(s3Client);
```

Comment:
pass the CitiBikeService into StationsCache as well, because you end up creating the CitiBikeService twice.